scroll-timeline-base.js

```diff
@@ -226,17 +224,11 @@ export class ScrollTimeline {
 
     // Step 3
     if (currentScrollOffset < startOffset) {
-      if (this.fill == 'none' || this.fill == 'forwards')
-        return unresolved;
       return 0;
```

Comment:
This is not quite consistent with the resolution we arrived at - because 0 will leave the animation in the active phase whereas it should be just before the active phase. Perhaps for the polyfill we should return -epsilon? Note that we can actually return a different currentTime than the one we assign to the internal animation currentTime.